warehouse/templates/packaging/detail.html

```diff
@@ -244,7 +244,8 @@ <h2 class="no-top-padding">Reverse Dependencies (TODO)</h2>
             <div class="release{{ ' release--latest' if loop.first else ' release--oldest' if loop.last }}{{ ' release--current' if is_current }}">
               <div class="release__meta">
                 {% if is_current %}
-                <p class="release__version">{{ hrelease.version }}</p>
+                <p class="release__version"><a href="{{ request.route_path('packaging.project', name=release.project.name) }}">
+                {{ hrelease.version }}</a></p>
```

Comment:
Oh, I see! I thought the version number was unnecessary because it was the latest. I will add it and also make sure my branch is updated. Thanks for the review!